README.md

```diff
@@ -1,18 +1,115 @@
+   Get the latest stable 'static' release.
+- [mkvtoolnix](https://mkvtoolnix.download/downloads.html#windows), the download page for Windows is [here](https://www.fosshub.com/MKVToolNix.html)  
+    Download the latest portable version. It's packaged as a .7z file, I'd recommend using [7zip](http://www.7-zip.org/download.html) to open these files
+- [mpv](https://mpv.io/), the download page for Windows is [here](https://mpv.srsfckn.biz/)  
+    Get the latest version
+- [MP4v2](https://code.google.com/archive/p/mp4v2/)  
+    Functional Windows binaries for this are hard to find, I've been using [this one](http://forum.doom9.org/showthread.php?t=171038). If anyone knows of a more official binary, please let me know
+- [RubyInstaller](https://rubyinstaller.org/downloads/)
+
+1. Download everything from the links above
+1. Run the Ruby installer and make sure "Add to path" is ticked during install
+1. Create `C:\bin` and [add it to your PATH](#adding-a-folder-to-your-path)
+1. From the HandBrake zip file extract `HandBrakeCLI.exe` and the `fonts\` directory to `C:\bin`
+1. From the ffmpeg zip file extract `bin\ffmpeg.exe` to `C:\bin`
+1. From the MKVToolNix 7z file extract `mkvpropedit` and the `data\` directory to `C:\bin`
+1. From the MP4v2 7z file extract `libmp4v2.dll` and `mp4track.exe` to `C:\bin`
+1. From the mpv 7z file extract `mpv.exe` and `d3dcompiler_43.dll` to `C:\bin`  
+    `C:\bin` should now contain:
+    - data\
+    - fonts\
+    - d3dcompiler_43.dll
+    - ffmpeg.exe
+    - HandBrakeCLI.exe
+    - libmp4v2.dll
+    - mkvpropedit.exe
+    - mp4track.exe
+    - mpv.exe
+1. Open `cmd.exe` and run `gem install video_transcoding`
+
+### Adding a folder to your PATH
+1. Right click `Start`
+1. Open Control Panel
+1. Search for "Path"
+1. Click "Edit System Environment Variables"
+1. Click the "Environment Variables" button in the lower right corner
+1. In the System Variables box find "PATH" and select it
+1. Click the "Edit" button
+
+## Further information
+### Batch control in `cmd.exe`
+Batch control can be achieved with the following .bat file:
+```
+@echo off
+
+setlocal EnableDelayedExpansion
+
+set work=%cd%
+set queue=%work%\queue.txt
+set crops=%work%\Crops
+
+for /F "tokens=*" %%I in (queue.txt) do (
+    set title_name=%%~nI
+    set crop_file=%crops%\!title_name!.txt
+
+    if exist !crop_file! (
+        for /F "tokens=* USEBACKQ" %%F in (`type "!crop_file!"`) do set crop_option=--crop %%F
+    ) else (
+        set crop_option=
+    )
+
+    call transcode-video !crop_option! "%%I"
+)
+```
+
+It works in much the same way that Don's [bash script](https://github.com/donmelton/video_transcoding#batch-control-for-transcode-video) works, although it does have the limitation of not being able to "resume" the queue in the way that Don's does. I would recommend creating a `batch-transcode.bat` file in your `C:\bin` folder, then you can create the necesarry file structure anywhere:
```

Comment:
Don's shell script modifies the queue.txt to remove items it's already encoded, but I'm not aware of an easy built in way to do that in a batch file